### PR TITLE
[frontend] Bulk creation for Channel entity (#4352)

### PR DESCRIPTION
**Pull request by lndrtrbn**
### Proposed changes

* UI components to manage bulk creation
* Possibility to add Channels in bulk

### Related issues

* #4352 

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case (coverage and e2e)
- [ ] I added/update the relevant documentation (either on github or on notion)
- [x] Where necessary I refactored code to improve the overall quality

### Further comments

Chunk 1 of the feature https://github.com/OpenCTI-Platform/opencti/issues/4352

### CODE REVIEW

File: opencti-platform/opencti-front/src/components/fields/BulkTextField/BulkTextField.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { FieldProps } from 'formik';
+import { Alert, Typography } from '@mui/material';
+import TextField from '../../TextField';
+import { useFormatter } from '../../i18n';
+
+// TODO make TextField component as typescript and reuse its props
```
Comment (lndrtrbn) [inline opencti-platform/opencti-front/src/components/fields/BulkTextField/BulkTextField.tsx:8]: FYI this todo was not expected to be resolved in this PR

File: opencti-platform/opencti-front/src/private/components/arsenal/channels/ChannelCreation.tsx [hunk 2]
```diff
@@ -156,90 +158,114 @@ export const ChannelCreationForm: FunctionComponent<ChannelFormProps> = ({
...
+              formValue={values.name}
+            />
+          )}
+          <Form style={{ margin: '20px 0 20px 0' }}>
+            <Field
+              component={isFeatureEnable('BULK_ENTITIES') ? BulkTextField : TextField}
+              name="name"
+              label={t_i18n('Name')}
+              fullWidth={true}
+              detectDuplicate={['Channel', 'Malware']}
```
Comment (aHenryJard) [inline opencti-platform/opencti-front/src/private/components/arsenal/channels/ChannelCreation.tsx:181]: why do we check duplicates on malware for channel ?

**Review by aHenryJard — APPROVED** [review]
Tested locally it's all fine with and without feature flag.

File: opencti-platform/opencti-front/src/private/components/arsenal/channels/ChannelCreation.tsx (hunk 2 above)
Comment (lndrtrbn) [reply opencti-platform/opencti-front/src/private/components/arsenal/channels/ChannelCreation.tsx:181]: I don't know, it was here before my work, and it is also the case for other threats entities

**Pull request merged**